RELEASE-NOTES.txt

```diff
@@ -1,11 +1,12 @@
 Unreleased
 ---
+* [**] Tapping on all nested blocks gets focus directly instead of having to tap multiple times depeding on the nesting levels. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5781]
 
 1.95.0
 ---
 * [**] Add fullscreen embed preview to Android [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5743]
 * [*] Fix crash when trying to convert to regular blocks an undefined/deleted reusable block [https://github.com/WordPress/gutenberg/pull/50475]
-* [**] Tapping on a nested block now gets focus directly instead of having to tap multiple times depending on the nesting levels. [https://github.com/WordPress/gutenberg/pull/50108]
+* [**] Tapping on nested text blocks gets focus directly instead of having to tap multiple times depeding on the nesting levels. [https://github.com/WordPress/gutenberg/pull/50108]
```

Comment:
There is a typo in the current release note.
```suggestion
* [**] Tapping on nested text blocks gets focus directly instead of having to tap multiple times depending on the nesting levels. [https://github.com/WordPress/gutenberg/pull/50108]
```

Reply:
An alternative to consider.
```suggestion
* [**] Tapping nested text block moves focus to the nested block directly, rather than requiring multiple taps to navigate down each nesting levels. [https://github.com/WordPress/gutenberg/pull/50108]
```